src/test/obj_realloc/obj_realloc.c

```diff
@@ -113,7 +151,7 @@ fill_buffer(unsigned char *buf, size_t size)
  */
 static void
 test_realloc(PMEMobjpool *pop, size_t size_from, size_t size_to,
-		unsigned type_from, unsigned type_to, int zrealloc)
+		unsigned type_from, unsigned long long type_to, int zrealloc)
```

Comment:
pmemobj_realloc uses uint64_t for type numbers, so you should use this type too

type_from should probably be changed too

Reply:
Done.